bonsai/ethereum-relay/src/sdk/client.rs

```diff
@@ -97,10 +97,10 @@ impl Client {
     }
 
     /// Construct a [Client] from url + api key strings
-    pub fn from_parts(api_url: String, api_key: String) -> Result<Self, ClientError> {
+    pub fn from_parts(api_url: String, api_key: &str) -> Result<Self, ClientError> {
```

Comment:
did you intend to also change `api_url` to `&str`?

Reply:
Yes, it's not consumed but only need to be borrowed.

Reply:
I meant that in this `from_parts` function the parameter `api_url` is a `String`. To stay consistent across the library I would prefer if this also was a `&str`.